js/dropdowns.js

```diff
@@ -41,7 +41,6 @@ function Dropdown(selector, opts) {
     }
   }
 
-  $(document.body).on('tag:removed', this.handleRemoveClick.bind(this));
```

Comment:
I don't remember what the original purpose was, but with changing the functionality of tags this resulted in a bug that removed the checkbox entirely when a tag was removed.